Makefile

```diff
@@ -0,0 +1,39 @@
+# Also needs to be updated in galaxy.yml
+VERSION = 1.0.0
+
+clean:
+	rm -f community-kubernetes-${VERSION}.tar.gz
+	rm -rf ansible_collections
+
+build: clean
+	ansible-galaxy collection build
+
+release: build
+	ansible-galaxy collection publish community-kubernetes-${VERSION}.tar.gz
+
+install: build
+	ansible-galaxy collection install -p ansible_collections community-kubernetes-${VERSION}.tar.gz
+
+test-sanity:
+	ansible-test sanity -v --docker --color $(TEST_ARGS)
+
+test-integration:
+	ansible-test integration --docker -v --color $(TEST_ARGS)
+
+test-molecule:
+	ansible-test integration --docker -v --color $(TEST_ARGS)
```

Comment:
This should be `molecule test`.

Also, can you update the `ci.yml` GitHub Actions workflow to use `make` to run the sanity/integration/molecule tests in CI? That way we're consistent in all the different cases.

Reply:
Absolutely! and yes ... I missed a spot there on that one, thanks!